recipes/spdlog/all/test_package/conanfile.py

```diff
@@ -13,5 +13,6 @@ def build(self):
 
     def test(self):
         if tools.cross_building(self.settings):
```

Comment:
This was added by #911 but I am not sure why the tests should only be run when cross building a package. 
Is there a particular reason for this?

Reply:
probably it was a mistake and it should be 
```suggestion
        if not tools.cross_building(self.settings):
```